ocs_ci/ocs/ui/views.py

```diff
@@ -1284,19 +1284,23 @@
         By.XPATH,
     ),
     "req_capacity_dropdown_selected": (
-        "//div[@id='breakdown-card-title']/following-sibling::*//*[@class = 'pf-c-select__toggle-text']",
+        "//div[@id='breakdown-card-title']/following-sibling::*//*[@class = 'pf-c-select__toggle-text'] | "
+        "//div[@class='pf-v5-c-select ceph-capacity-breakdown-card-header__dropdown']"
+        "//*[@class='pf-v5-c-select__toggle-text']",
         By.XPATH,
     ),
     "req_capacity_dropdown_btn_one": (
-        "//div[@class='pf-c-select ceph-capacity-breakdown-card-header__dropdown']",
+        "//div[@class='pf-c-select ceph-capacity-breakdown-card-header__dropdown'] | "
+        "//div[@class='pf-v5-c-select ceph-capacity-breakdown-card-header__dropdown']",
         By.XPATH,
     ),
     "req_capacity_dropdown_btn_two": (
-        "(//span[@class='pf-c-select__toggle-arrow'])[2]",
+        "(//span[@class='pf-c-select__toggle-arrow'])[2] | "
+        "(//span[@class='pf-v5-c-select__toggle-arrow'])[2]",
         By.XPATH,
     ),
     "req_capacity_dropdown_list_option": (
-        "//*[@class='pf-c-select__menu-item' and contains(text(), '{}')]",
+        "//button[contains(@class, 'c-select__menu-item') and contains(text(), '{}')]",
```

Comment:
good way to resolve issue with "pf-<number>"  class changes, even better than to use "or" in locator.
If it is possible pls apply this approach to all locators above. Will be really more elegant and hypothetically will work with Patternfly 6